content/docs/reference/pkg/nodejs/pulumi/packet/_index.md

```diff
@@ -27,7 +27,6 @@ import * as packet from "@pulumi/packet";
 <h3>Modules</h3>
 <ul class="api">
     <li><a href="config/"><span class="symbol module"></span>config</a></li>
-    <li><a href="types/"><span class="symbol module"></span>types</a></li>
```

Comment:
Looks like another case of #1685. @stack72, could you try regenerating the docs for `packet` again until this line isn't removed? It's really strange behavior in the doc generator -- sometimes it happens, sometimes it doesn't. Going to look into this strange bug today.